am-lib/src/main/java/uk/gov/hmcts/reform/amlib/utils/Permissions.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.amlib.utils;
+
+import uk.gov.hmcts.reform.amlib.enums.Permission;
+import uk.gov.hmcts.reform.amlib.exceptions.UnsupportedPermissionsException;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public final class Permissions {
+
+    private Permissions() {
```

Comment:
Sonar might be complaining about it (at least it was for config of my previous project in HMCTS). Please add a comment like "we don't want this class to be instantiated"

Reply:
👍

Reply:
Some people add `// NO-OP` comment to empty private constructors.